pydis_site/apps/content/resources/guides/python-guides/windows/common-issues.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Common Issues Using Python on Windows
+description: A list of common issues with Python on Windows and how to fix them
+icon: fab fa-windows
+toc: 3
+---
+
+### When I run `python` in the terminal I get no result, or the Windows Store opens!
+
+By default Windows has an alias for `python` in the terminal to guide you to install it from the [Microsoft
+Store](https://apps.microsoft.com/store/search/python). We don't recommend installing Python from the Microsoft
+Store, see why [here](../microsoft-store). To disable this alias, search "App execution aliases" in Windows search
+and click on "Manage app execution aliases". In the list you should see two options with title "App Installer" and
+descriptions `python.exe` and `python3.exe`. Disable both of them.
+
+Note that if you are following instructions telling you to run a command starting with `python3`, those instructions are
+intended for Unix/macOS systems. Try just using `python` instead.
+
+If after doing this you have an issue with the Python command not being detected, see the question below.
+
+### When I try and run my code with `python` in the terminal I get an error saying the command was not recognized
+
+If you have not installed Python, you will need to do that. Follow our guide [here](../installing-and-using-python).
+
+If you have installed python and are still having the issue, it is likely that you didn't check the "Add python.exe to
+PATH" checkbox on the first page of the installer. See [our guide on adding Python to PATH](../putting-python-on-path)
+for how to fix this. You can also use the `py` launcher instead of `python` by just replacing `python` in your command
+with `py`. See [this guide](../installing-and-using-python/#the-py-launcher) for more information on that.
+
+### I `pip` installed a package but when running my code get a `ModuleNotFoundError`
+
+-   #### Are you actually getting a `ModuleNotFoundError`?
+
+    If you are using a code editor such as VSCode or PyCharm you may get a squiggly line under your import saying the
+    module couldn't be found, it is possible that this is just an mistake by the editor, so try actually running your
+    code with to ensure if it actually errors.
+
+    If your code runs fine, you could try restarting your editor. If you have newly installed a module it may just not
+    have detected it yet. If that doesn't help, you may need to configure your editor to ensure it is looking for the
```

Comment:
Actually, some editors can still show that error even if you're showing the correct virtual environment with the installed package.
I don't know what the package was, but I remember this scenario so I just had to ignore it.

Reply:
Is that this https://stackoverflow.com/questions/34365044/pycharm-does-not-recognize-cv2-as-a-module ?
Where the fix is something like `from cv2 import cv2`. Could add a note to that SO link.